utilities/cwp/custom-war-packager.inc

```diff
@@ -0,0 +1,90 @@
+# function to download the CWP jar
+# $1: Working directory
+# $2: CWP version to use
+download_cwp() {
+
+	if [ "$#" -gt 1 ]
+	then
+		cwp_jar_file="$1/.cwp-build/cwp-cli-$2.jar"
+		if [ "$#" -eq 4 ]
+		then
+            image_tag="$4"
+        else
+            image_tag="jenkins-experimental/jenkinsfile-runner-test"
+        fi
+
+		rm -rf "$1/.cwp-build"
+		mkdir -p "$1/.cwp-build"
+		wget -O "$cwp_jar_file" https://repo.jenkins-ci.org/releases/io/jenkins/tools/custom-war-packager/custom-war-packager-cli/$2/custom-war-packager-cli-$2-jar-with-dependencies.jar
```

Comment:
It will work only for the released versions of CWP. Support of Incrementals/Snapshots could be added later though

Reply:
My intention was  to have a first PR so we can start working in parallel if needed. I agree the support for incrementals/snapshots can be added in a second PR based on this one.